assets/scss/components/_external-markup.scss

```diff
@@ -0,0 +1,27 @@
+/*
+External Markup
+
+Partial HTML fragments from other services may be vanilla HTML with no classes upon which to apply styles.
+This module can house any wrapper styles to apply to this content.
+
+Markup:
+<div class="{{modifier_class}}">
+  <h2>Heading level 2</h2>
+  <p>Paragraph text</p>
+  <a href="">Anchor</a>
+</div>
+
+.external-markup--xsmall - for extra small external markup
+
```

Comment:
I am going to raise an issue as i'd like other thoughts on it. 
`BEM` wise this is a modifier without a base class, `.external-markup` is missing. But the comments have been added in the same way `.external-markup--xsmall` was made to be "future proofed" for other examples.
Its tempting to call `YAGNI` on it and say `.external-markup--xsmall` should really be `.external-markup-xsmall`